lang.json

```diff
@@ -48,8 +48,7 @@
     ],
     "maturity": "alpha",
     "exts": [
-      ".c",
```

Comment:
This should not be part of the PR.
You should probably rebase on main (or develop, whatever the master branch is).

Reply:
not sure what can be done; the commit that added those filetypes added them to this autogenerated file and i autogenerated it away. Ill at a commit to this PR to add them properly to `generate.py`